site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>Use <code>and</code> instead of  <code>&amp;</code> - <a href="https://developers.google.com/style/text-formatting" target="_blank">https://developers.google.com/style/text-formatting</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>